### DISCUSSION TITLE
Compatibility with Chocolatey package manager

### DISCUSSION THREAD

**Discussion by iqgustavo** [espanso/espanso#1248] [Feature requests and Ideas]
**Is your feature request related to a problem? Please describe.**
I want to be able to install and update the software easily with a single command using chocolatey.org
An example of the problem and solution can be read here:
https://www.reddit.com/r/espanso/comments/k65goe/espanso_073_is_out/hr9u4f1/?context=3

Please please consider chocolatey support in the future ...
I simply open the command prompt and type "cup -y espanso"
... and espanso gets installed or updated to the latest version automatically on windows
... or I type "cup all" and all my apps installed with chocolatey get updated.

Just as a reference here I write the original already solved problem:
Dear Espanso developers,
Thanks a lot!!! ... I really loved your previous version... I just updated to 0.7.3 through chocolatey.org installer and the program no longer starts on my machine Win10 English with win10debloater ... tried reinstalling and I am unable to make it run again. Please how can I troubleshoot this?
UPDATE: SOLVED... I manually deleted all the appdata found with RevoUninstaller in diferent places and backedup my config in C:\Users\MYUSERNAME\AppData\Roaming\espanso/default.yml

**Describe the solution you'd like**
Work together with chocolatey team to support update compatibility.

**Describe alternatives you've considered**
Every update when the app stops working correctly I would have to:
1. Backup configuration file
2. Install RevoUninstaller
3. Uninstall espanso program
4. Search and delete left overs
5. Reinstall espanso
6. Replace configuration file

**Additional context**
Chocolatey webpage for espanso ... Thanks to this webpage I was able to install espanso.
https://community.chocolatey.org/packages/espanso

**Comment by federico-terzi** [comment]
Hey @iqgustavo,

Thank you for the feedback! I'll make sure to investigate Chocolatey support in the near future :)

Cheers :)
